watchtower-plugin/src/main.rs

```diff
@@ -295,6 +296,37 @@ async fn get_tower_info(
     }
 }
 
+async fn ping(
+    plugin: Plugin<Arc<Mutex<WTClient>>>,
+    v: serde_json::Value,
+) -> Result<serde_json::Value, Error> {
+    let (tower_net_addr, proxy) = {
+        // Check if the tower_id is known to the plugin
+        let tower_id = TowerId::try_from(v).map_err(|e| anyhow!(e))?;
+        let state = plugin.state().lock().unwrap();
+        (
+            state
+                .towers
+                .get(&tower_id)
+                .ok_or(anyhow!("Unknown tower_id"))?
+                .net_addr
+                .clone(),
+            state.proxy.clone(),
+        )
+    };
+    let response = get_request(&tower_net_addr, Endpoint::Ping, &proxy)
+        .await
+        .map_err(to_cln_error)?;
+
+    if response.status().is_success() {
+        log::debug!("Ping successful!");
+        Ok(json!("Tower is reachable"))
+    } else {
+        log::debug!("Ping failed with status code: {}", response.status());
+        Err(anyhow!("Tower cannot be reached"))
+    }
```

Comment:
It was me actually suggesting this so the user UX was better (especially for the error case where he was previously formatting with debug).

It may be OK to merge this. Any suggestions @mariocynicys?

Reply:
> It was me actually suggesting this so the user UX was better

You mean the not reporting the failure status code in here? I think it's gonna be useful for the user. It's an error anyways so that's an already frustrating experience :joy:.

If it for the log lines, I can't think of a reason why the first one would be useful. The second one though might be useful as a record of when the tower (or us) went offline/unresponsive/etc... but then I thought there would probably be a similar log line from the retrier or something that states the same.

Reply:
Yeah, I mean for returning simpler messages to standard output.

Maybe something like this would be better:

```suggestion
    if response.status().is_success() {
        Ok(json!("Tower is reachable"))
    } else {
        Err(anyhow!(format!("Tower cannot be reached (Error code: {})", response.status())))
    }
```